src/program-spec.ts

```diff
@@ -100,11 +102,7 @@ function Spec<Data, Inputs extends Record<string, Input>>(
   return { inputs, logic: { assert, data } };
 }
 
-const Input = {
-  claim,
-  private: privateParameter,
-  constant,
-};
```

Comment:
@martonmoro I decided to remove `Input.private()` for now since it doesn't have a place in the spec. I replaced it with `Credential.none()` in all the tests, which is another way of getting arbitrary private data into the circuit